example.groovy

```diff
@@ -0,0 +1,5 @@
+class Example {
```

Comment:
📝 **[npm-groovy-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class should be marked with one of @GrailsCompileStatic, @CompileStatic or @CompileDynamic  CompileStatic